### PR TITLE
0.1.1 fix find lib

### PR DESCRIPTION
**Pull request by turboladen**
Branches off of the `0.1.1` tag and fixes the `find_lib` function. Also adds some docker stuff to allow for testing both GDAL 3 and GDAL 2.

I also created a `0.1-current` branch in case we need to do any more dev with the 0.1 line. As such, this PR targets that.

- [x] squash

### CODE REVIEW

**Comment by tindron** [comment]
Can you pull the `.github` dir into this branch as well so the specs run?

**Review by roaming-rockenfels — APPROVED** [review]

**Comment by turboladen** [comment]
> Can you pull the `.github` dir into this branch as well so the specs run?

done.

**Review by tindron — APPROVED** [review]

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
```
Comment (tindron) [inline .github/workflows/continuous-integration.yml]: Should we be testing against all of the ruby versions our services run on?
```suggestion
    strategy:
      fail-fast: false
      matrix:
        ruby: [2.6, 2.7, '3.0', 3.1]
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v2
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby }}
          bundler-cache: true
```

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y libgdal-dev liblwgeom-dev
+          bundle install --jobs 4 --retry 3
+      - name: Run tests
+        run: bundle exec rake spec
```
Comment (tindron) [inline .github/workflows/continuous-integration.yml]: If you add rspec-github as a dev dependency you can
```suggestion
        run: bundle exec rspec --format RSpec::Github::Formatter --format progress
```

**Review by tindron — APPROVED** [review]

**Pull request merged**